server/src/test/java/org/opensearch/action/search/PitTestsUtil.java

```diff
@@ -128,4 +133,10 @@ public static void assertGetAllPitsEmpty(Client client) throws ExecutionExceptio
         GetAllPitNodesResponse getPitResponse = execute1.get();
         Assert.assertEquals(0, getPitResponse.getPitInfos().size());
     }
+
+    public static void assertSegments(boolean isEmpty, Client client) {
+        IndicesSegmentResponse indicesSegmentResponse = client.execute(PitSegmentsAction.INSTANCE, new PitSegmentsRequest()).actionGet();
+        assertTrue(indicesSegmentResponse.getShardFailures() == null || indicesSegmentResponse.getShardFailures().length == 0);
+        assertEquals(indicesSegmentResponse.getIndices().isEmpty(), isEmpty);
```

Comment:
Don't we need to test if `indicesSegmentResponse` is as expected? isEmpty() check is not full-proof.

Reply:
Added index name and number of shards as well from the result.